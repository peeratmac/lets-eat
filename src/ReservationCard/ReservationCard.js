import React from 'react';

const ReservationCard = ({ date, id, name, number, time }) => {
  return (
    <div>
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time}</p>
      <p>Guests: {number}</p>
      <button>Cancel</button>
    </div>
  );
};

export default ReservationCard;
