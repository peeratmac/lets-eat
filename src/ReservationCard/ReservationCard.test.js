import React from 'react';
import { shallow } from 'enzyme';
import ReservationCard from './ReservationCard';

describe('RESERVATION CARD', () => {
  it('should match the ReservationCard Component Snapshot', () => {
    const wrapper = shallow(<ReservationCard />);

    expect(wrapper).toMatchSnapshot();
  });
});
