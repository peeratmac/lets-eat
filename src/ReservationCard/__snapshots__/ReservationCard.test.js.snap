// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RESERVATION CARD should match the ReservationCard Component Snapshot 1`] = `
<div>
  <h2 />
  <p />
  <p />
  <p>
    Guests: 
  </p>
  <button
    onClick={[Function]}
  >
    Cancel
  </button>
</div>
`;
